Packs/DarktraceEventCollector/ModelingRules/DarktraceEventCollector/DarktraceEventCollector.xif

```diff
@@ -0,0 +1,43 @@
+[MODEL: dataset ="darktrace_darktrace_raw"]
+alter
+    xdm.alert.description = json_extract_scalar(model, "$.then.description"),
+    xdm.event.description = json_extract_scalar(model, "$.then.description"),
+    xdm.alert.mitre_tactics = arraymap(json_extract_array(model, "$.then.mitre.tactics"), trim("@element", "\"")),
+    xdm.alert.mitre_techniques = arraymap(json_extract_array(model, "$.then.mitre.techniques"), trim("@element", "\"")),
+    xdm.alert.original_alert_id = to_string(pbid),
```

Comment:
wasn't this supposed to be changed to event_id?

Reply:
We added them as extra fields according to Chanan.